.github/CODEOWNERS

```
@@ -4,4 +4,4 @@
 # See: https://help.github.com/articles/about-codeowners/
 
 # These owners will be the default owners for everything in the repo.
-*       @daneshk @niveathika @kaneeldias
+*       @daneshk @shafreenAnfar @lnash94
```

Better to add the primary owner as the first entry. That's the convention we've been following for the other modules as well
```suggestion
*        @lnash94 @daneshk @shafreenAnfar
```